src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getSum = (a) => {
+    const add = (b) => {
+      if (count === 3) {
+        count--;
+
+        return 'Bzzz... Error!';
+      }
+
+      count++;
```

Comment:
You should increment the count before the condition check, not after it. Move this line to the beginning of the function.